source/includes/APIReference/CustomFieldItems/Examples/_retrieve.tmpl.erb

```diff
@@ -32,3 +32,12 @@
     :response_body => @response_body
```

Comment:
Remove `response_body` param.  Only include in the last example (your new one).

Reply:
Done